receiver/jaegerreceiver/factory_test.go

```diff
@@ -275,15 +275,15 @@ func TestRemoteSamplingFileRequiresGRPC(t *testing.T) {
 
 func TestCustomUnmarshalErrors(t *testing.T) {
 	factory := Factory{}
-	v := viper.New()
```

Comment:
when this change gets to contribs we will have to check for these usages too.

Reply:
+1